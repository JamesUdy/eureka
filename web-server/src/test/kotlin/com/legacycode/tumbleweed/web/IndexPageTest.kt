package com.legacycode.tumbleweed.web

import org.approvaltests.Approvals
import org.junit.jupiter.api.Test

class IndexPageTest {
  @Test
  fun `replace port number in an index HTML file`() {
    // given
    val indexPage = IndexPage.withPort(7997, "test-index-page.html")

    // when & then
    Approvals.verify(indexPage.content)
  }
}
