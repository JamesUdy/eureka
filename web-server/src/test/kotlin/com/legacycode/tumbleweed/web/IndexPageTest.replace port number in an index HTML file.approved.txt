<html>
  <head>
    <title>Test HTML</title>
  </head>
  <body>
    <div>localhost:7997</div>
  </body>
</html>
